tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -15,8 +15,6 @@
 *** Settings ***
 Documentation  Test 10-01 - VCH Restart
 Resource  ../../resources/Util.robot
-Suite Setup  Install VIC Appliance To Test Server
```

Comment:
I would prefer to keep these here and override them for the specific tests using the following:
```
[Setup]  Install VIC Appliance ...
...
[Teardown]  Cleanup VIC Appliance On Test Server
```

This would mean that `[Teardown]` doesn't need to be used as it's common for all tests, and it's only the open network test that needs to override `[Setup]`